README.md

```diff
@@ -35,7 +35,8 @@ wgpu_profiler!("name of your scope", &mut profiler, &mut encoder, &device, {
   // wgpu commands go here
 });
 ```
-Unless you disable timer scoping (`wgpu_profile` will still emit debug scopes), your wgpu device needs `wgpu::Features::TIMESTAMP_QUERY` enabled.
+Note that `GpuProfiler` reads the device features - if your wgpu device doesn't support `wgpu::Features::TIMESTAMP_QUERY`, it will automatically not attempt to emit any timer queries.
```

Comment:
> if your wgpu device doesn't support `wgpu::Features::TIMESTAMP_QUERY`

In this snippet, I prefer the wording "doesn't have `wgpu::Features::TIMESTAMP_QUERY` enabled" because it more strongly suggests that this is something the user needs to explicitly enable when creating the device.